### DISCUSSION TITLE
Fill variables in airflow instance automaticly

### DISCUSSION THREAD

**Discussion by waardd** [apache/airflow#30707] [Q&A]
We have 4 different environments (5 if counting a sandbox).
Can we fill the Variables within airflow with entries in a automated way?

**Comment by potiuk** [comment]
Sure. You can use airflow CLI https://airflow.apache.org/docs/apache-airflow/stable/cli-and-env-variables-ref.html#variables with export/import or airflow  REST API (https://airflow.apache.org/docs/apache-airflow/stable/stable-rest-api-ref.html#tag/Variable). Both are intended for automation.
